docs/reference/aggregations/bucket/significantterms-aggregation.asciidoc

```diff
@@ -385,6 +385,100 @@ Google normalized distance as described in https://arxiv.org/pdf/cs/0412098v3.pd
 // NOTCONSOLE
 `gnd` also accepts the `background_is_superset` parameter.
 
+[role="xpack"]
+[[p-value-score]]
+===== p-value score
+
+[NOTE]
+====
+To use this heuristic, you must
+have the {subscriptions}[appropriate subscription] for {ml-features}.
```

Comment:
I am not sure how to best word this. It's "basic". So, it is free if the ML plugin is enabled. 

So, they just need to have the ML plugin enabled. I am not sure about "subscription"

Reply:
Can you clarify how to enable the plugin? We currently only describe how to add ML nodes, but this seems like something else.

Reply:
@lcawl , it just means that the plugin is part of the distribution. There is a way to disable the plugin by setting `xpack.ml.enabled` to `false`. You are talking about nodes that are valid places to run ML jobs (the node role setting).

Reply:
In this case, my suggestion is that we remove this "note" but leave the role=xpack and make sure the p-score functionality is added to the subscriptions page in the appropriate subscription tier.

Reply:
I removed the note via f7255d5